build.sbt

```diff
@@ -53,10 +51,7 @@ lazy val commonSettings = Seq(
       "-groups",
       "-sourcepath", (baseDirectory in LocalRootProject).value.getAbsolutePath,
       "-doc-source-url", "https://github.com/typelevel/discipline-scalatest/blob/v" + version.value + "€{FILE_PATH}.scala"
-  ),
-
-  addCompilerPlugin("org.typelevel" %% "kind-projector"     % kindProjectorV),
-  addCompilerPlugin("com.olegpy"    %% "better-monadic-for" % betterMonadicForV),
```

Comment:
Why remove better-monadic-for? Do we literally just not have for-comprehensions at all? In general I like to have this plugin just because it removes a major gotcha from the compiler.